### PR TITLE
318 invalid moab status

### PR DESCRIPTION
**Pull request by SaravShah**
- want to distinguish between objects failing moab validation and objects failing checksum validation.

connects #318

### CODE REVIEW

**Comment by SaravShah** [comment]
I was trying to split it into two commits, but i just added it!

**Comment by SaravShah** [comment]
fixed @jmartin-sul

File: .rubocop.yml [hunk 1]
```diff
@@ -73,6 +73,7 @@ Metrics/LineLength:
     - 'spec/services/preserved_object_handler_spec.rb'
     - 'spec/services/preserved_object_handler_update_version_spec.rb'
     - 'spec/services/shared_examples_preserved_object_handler.rb'
+    - 'spec/models/status_spec.rb' # 132 - better readability
```
Comment (ndushay) [inline .rubocop.yml]: what does 132 refer to?  a line number?  If so, maybe indicate?

**Comment by SaravShah** [comment]
fixed it! @ndushay

**Pull request merged**